kubetest/e2e.go

```diff
@@ -506,16 +518,21 @@ func KubemarkTest(dump string, o options) error {
 	})
 
 	// Start new run
-	if popN, err := pushEnv("NUM_NODES", o.kubemarkNodes); err != nil {
+	if popN, err := pushEnv("NUM_NODES", nodes); err != nil {
 		return err
 	} else {
 		defer popN()
 	}
-	if popM, err := pushEnv("MASTER_SIZE", o.kubemarkMasterSize); err != nil {
+	if popM, err := pushEnv("MASTER_SIZE", masterSize); err != nil {
```

Comment:
I guess we can totally get rid of MASTER_SIZE env var as it should be calculated automatically based on NUM_NODES. See - https://github.com/kubernetes/kubernetes/blob/master/cluster/kubemark/gce/config-default.sh#L30